homeassistant/components/media_player/__init__.py

```diff
@@ -66,6 +66,18 @@
 
 SCAN_INTERVAL = timedelta(seconds=10)
 
+DEVICE_CLASS_MEDIA = 'media'
```

Comment:
Let's open an architecture issue for this. We should also make sure that this PR is accompanied with a PR for the developer docs to document them.